virtualization.go

```diff
@@ -7,12 +7,24 @@ package vz
 */
 import "C"
 import (
+	"errors"
 	"runtime"
 	"runtime/cgo"
+	"strconv"
+	"strings"
 	"sync"
+	"syscall"
 	"unsafe"
 )
 
+// ErrUnsupportedOSVersion is returned when calling a method which is only
+// available in newer macOS versions.
+// This allows for example to build an application using the file sharing API,
+// and to be able to run it both on macOS 11 and macOS 12. When running on
+// macOS 11, the file sharing API will return ErrUnsupportedOSVersion, while on
+// macOS 12, the file sharing API will work as expected.
```

Comment:
```suggestion
```

Reply:
Added such an example, and shortened the doc comment.